test/fprime/fbuild/test_settings.py

```diff
@@ -39,6 +39,7 @@ def test_settings():
                 "ac_constants": full_path("..") / "config" / "AcConstants.ini",
                 "project_root": full_path(".."),
                 "config_directory": full_path("..") / "config",
+                "default_cmake_options": "",
```

Comment:
Should one of these tests provide a "default_cmake_options" with actual values instead of the empty string?

Reply:
I will fix this one!  We should test the multi-line setup.

Reply:
Added a new test case!